final_html_output.py

```diff
@@ -32,3 +32,11 @@ def final_html_output(df: pd.DataFrame) -> str:
 
 def link_username(u: str):
     return f'<a target="_blank" rel="noopener noreferrer" href="https://github.com/{u}">{u}</a>'
+
+
+def add_user_avatar(df: pd.DataFrame) -> pd.DataFrame:
+    image_tag = lambda u: f'<img src="https://github.com/{u}.png?size=20" alt="@{u}" />'
+    avatarUrls = df["User Name"].apply(image_tag)
+    df.insert(0, "Avatar Url", avatarUrls)
```

Comment:
@shresthalucky We don't need to add the avatar to the data frame. In fact, the avatar has nothing to do with the data frame. We should separate this concern.

Instead what we can do is change `link_username` function to `format_username` and use the username --> `u: str` directly and create the intended html.

Sorry to nag you here a little bit. Hope it makes sense. 😅

Reply:
Also, the avatar and the username are too far out. Can we put them close like what GitHub does.

<img width="105" alt="Screenshot 2023-10-10 at 3 36 41 PM" src="https://github.com/Lf-Network/oss-leaderboard/assets/4240991/440cc7af-0a57-4a00-81d9-8dd960de1f18">

We should probably also add class names in the elements and maybe wrap avatar + username in a div so that we can target it properly instead of traversing through the table in our CSS.

Reply:
One last thing, we should wrap the image with an ahref as well just like GitHub. So, clicking on the username or the username will take us to the user's GitHub account.

Reply:
@mesaugat It totally makes sense. Thank you. I've updated the PR. :smiley:

Reply:
Reverting the css changes.